File: Assignment-40/Program3.java
import  java.util.*;

class Program3
{
      public static void main(String arg[])   
      {
            Pattern pobj = new Pattern();
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter numbers of rows :");
            int i = sobj.nextInt();

            System.out.println("Enter numbers of colum :");
            int j = sobj.nextInt();

            pobj.Display(i,j);
      }
}
class Pattern
{
      public void Display(int iRow ,int iCol)
      {
            for(int i = 1;i<=iRow;i++)
            {
                  for(int j = iCol;j>=1;j--)
                  {
                       if(((j==1)||(j==iCol))||((i==1)||(i==iRow)))
                        {
                              System.out.print("*\t");
                             
                        }
                        else if(j==i)
                        {
                              System.out.print("*\t");
                        }
                        else 
                        {
                              System.out.print(" \t");
                        }
                       
                         
                        
                       
                  }
                  System.out.println();
            }
      }
}

File: Assignment-20/program3.c
#include<stdio.h>

void Display(int iRow,int iCol)
{
      int i = 0 ,j = 0;
      char ch  = 'A';

      for(i  = 1;i<=iRow;i++)
      {
            for(j = 1;j<=iCol;j++)
            {
                  printf("%c\t",ch);
                  
            }
            ch++;
            printf("\n");
      }     
     
}
int main()
{
      int ivalue1 = 0;
      int ivalue2 = 0;

      printf("Enter The Nunmber Row :");
      scanf("%d",&ivalue1);

      printf("Enter The Nunmber Row :");
      scanf("%d",&ivalue2);


  
      Display(ivalue1,ivalue2);

      return 0;
}